README.md

```diff
@@ -1214,7 +1215,7 @@ A STATIC_FILTER allows you to filter on a custom specified field, while a RADIUS
 {    
   options: [
     {
-      // Required, the value of the radius to apply (in meters). If this value is 0, no explicit locationRadius parameter will be applied to the request.
+      // Required, the value of the radius to apply (in meters). If this value is 0, the SDK will not add explicit radius filtering to the request.
```

Comment:
Might be good to quickly mention that the backend (?) might still do some distance filtering of it's own

Reply:
sure, added a blurb. I wanted to avoid saying specifically what the backend does because 1. I don't know and 2. they're free to change whenever they like.